### DISCUSSION TITLE
after updating from 1.9rc3 to 1.10.0 SSD is not recognized

### DISCUSSION THREAD

**Discussion by pirafal** [raspiblitz/raspiblitz#4362] [Q&A]
hello,
after updating from 1.9rc3 to 1.10.0 SSD is not recognized.

anyone have an idea?

RPI4 4GB

/home/admin/config.scripts/network.txindex.sh: line 10: /mnt/hdd/raspiblitz.conf: No such file or directory

sudo ls -al /mnt/hdd/:

total 6936592
drwxr-xr-x 2 root root       4096 Jan 12 17:00 .
drwxr-xr-x 4 root root       4096 Jan 12 14:37 ..
-rw-r--r-- 1 root root         10 Jan 12 16:57 .conf
-rw------- 1 root root 7103053824 Jan 12 17:00 swapfile

sudo df -h:

Filesystem      Size  Used Avail Use% Mounted on
/dev/root        29G   22G  6.7G  77% /
devtmpfs        1.9G     0  1.9G   0% /dev
tmpfs           1.9G     0  1.9G   0% /dev/shm
tmpfs           769M 1012K  768M   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs            32M     0   32M   0% /var/cache/raspiblitz
/dev/mmcblk0p1  255M   32M  224M  13% /boot
tmpfs           385M   16K  385M   1% /run/user/1000
tmpfs           385M   16K  385M   1% /run/user/117

**Comment by rootzoll** [comment]
What does the command `lsblk` output?
